<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HamiltonianHeisenbergPairsWidget</class>
 <widget class="QWidget" name="HamiltonianHeisenbergPairsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>1160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="2">
    <widget class="QScrollArea" name="scrollArea_5">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>376</width>
        <height>1136</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_29">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="14" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="8" column="0">
          <layout class="QGridLayout" name="gridLayout_40">
           <item row="2" column="0">
            <spacer name="horizontalSpacer_32">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_15">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Direction (x,y,z)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>mu_spin</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="11" column="1">
          <layout class="QGridLayout" name="gridLayout_47">
           <item row="0" column="0">
            <widget class="QLineEdit" name="lineEdit_anix_aniso">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_aniy_aniso">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLineEdit" name="lineEdit_aniz_aniso">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <spacer name="horizontalSpacer_35">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBox_aniso_periodical_a">
           <property name="text">
            <string>Periodical a</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QCheckBox" name="checkBox_ani_aniso">
           <property name="text">
            <string>Anisotropy</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="8" column="1">
          <layout class="QGridLayout" name="gridLayout_32">
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_extHx_aniso">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="lineEdit_extHy_aniso">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <spacer name="horizontalSpacer_24">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="3">
            <widget class="QLineEdit" name="lineEdit_extHz_aniso">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_muSpin_aniso">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="6" column="1">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="15" column="0">
          <widget class="QCheckBox" name="checkBox_dmi">
           <property name="text">
            <string>DMI</string>
           </property>
          </widget>
         </item>
         <item row="17" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_ddi">
           <property name="maximum">
            <double>1000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="checkBox_aniso_periodical_c">
           <property name="text">
            <string>Periodical c</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="checkBox_aniso_periodical_b">
           <property name="text">
            <string>Periodical b</string>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="16" column="1">
          <spacer name="verticalSpacer_7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="10" column="1">
          <layout class="QGridLayout" name="gridLayout_42">
           <item row="0" column="0">
            <widget class="QLineEdit" name="lineEdit_ani_aniso">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <spacer name="horizontalSpacer_29">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="17" column="0">
          <widget class="QCheckBox" name="checkBox_ddi">
           <property name="text">
            <string>Dipole-Dipole radius</string>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QCheckBox" name="checkBox_exchange">
           <property name="text">
            <string>Exchange</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <layout class="QGridLayout" name="gridLayout_46">
           <item row="0" column="1">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Direction (x,y,z)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <spacer name="horizontalSpacer_36">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="7" column="2">
          <spacer name="horizontalSpacer_31">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="7" column="0">
          <widget class="QCheckBox" name="checkBox_extH_aniso">
           <property name="text">
            <string>External Field</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <layout class="QGridLayout" name="gridLayout_36">
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_extH_aniso">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer_27">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="13" column="1">
          <layout class="QGridLayout" name="gridLayout_exchange">
           <item row="0" column="1">
            <widget class="QSpinBox" name="spinBox_nshells_exchange"/>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Number of Shells</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="14" column="1">
          <spacer name="verticalSpacer_9">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="15" column="1">
          <layout class="QGridLayout" name="gridLayout_dmi">
           <item row="0" column="1">
            <widget class="QSpinBox" name="spinBox_nshells_dmi"/>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Number of Shells</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="15" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="28" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="23" column="0">
        <widget class="QPlainTextEdit" name="plainTextEdit_pairs">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="plainText">
          <string># Manual pair input
i   j   da  db  dc   Jij   Dx  Dy  Dz
</string>
         </property>
        </widget>
       </item>
       <item row="21" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Pairs</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButton_pairs_apply">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="pushButton_pairs_fromfile">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>From File</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>30</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="27" column="0">
        <widget class="QPlainTextEdit" name="plainTextEdit_quadruplets">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="plainText">
          <string># Manual quadruplet input
i  j  k  l   da_j  db_j  dc_j   da_k  db_k  dc_k   da_l  db_l  dc_l   Q</string>
         </property>
        </widget>
       </item>
       <item row="24" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="26" column="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButton_quadruplets_apply">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Apply</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="pushButton_quadruplets_fromfile">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>From File</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Quadruplets</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>30</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="12" column="0">
        <layout class="QGridLayout" name="gridLayout_37">
         <item row="0" column="0">
          <widget class="QLabel" name="label_32">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Apply to</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_26">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_Hamiltonian_Ani_ApplyTo">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Current Image</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Current Image Chain</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>All Images</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="13" column="0">
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea_5</tabstop>
  <tabstop>comboBox_Hamiltonian_Ani_ApplyTo</tabstop>
  <tabstop>checkBox_aniso_periodical_a</tabstop>
  <tabstop>checkBox_aniso_periodical_b</tabstop>
  <tabstop>checkBox_aniso_periodical_c</tabstop>
  <tabstop>checkBox_extH_aniso</tabstop>
  <tabstop>lineEdit_extH_aniso</tabstop>
  <tabstop>lineEdit_muSpin_aniso</tabstop>
  <tabstop>lineEdit_extHx_aniso</tabstop>
  <tabstop>lineEdit_extHy_aniso</tabstop>
  <tabstop>lineEdit_extHz_aniso</tabstop>
  <tabstop>checkBox_ani_aniso</tabstop>
  <tabstop>lineEdit_ani_aniso</tabstop>
  <tabstop>lineEdit_anix_aniso</tabstop>
  <tabstop>lineEdit_aniy_aniso</tabstop>
  <tabstop>lineEdit_aniz_aniso</tabstop>
  <tabstop>checkBox_exchange</tabstop>
  <tabstop>spinBox_nshells_exchange</tabstop>
  <tabstop>checkBox_dmi</tabstop>
  <tabstop>spinBox_nshells_dmi</tabstop>
  <tabstop>checkBox_ddi</tabstop>
  <tabstop>doubleSpinBox_ddi</tabstop>
  <tabstop>pushButton_pairs_apply</tabstop>
  <tabstop>pushButton_pairs_fromfile</tabstop>
  <tabstop>plainTextEdit_pairs</tabstop>
  <tabstop>pushButton_quadruplets_apply</tabstop>
  <tabstop>pushButton_quadruplets_fromfile</tabstop>
  <tabstop>plainTextEdit_quadruplets</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
